practice/007/work/impl.ml

```diff
@@ -0,0 +1 @@
+let flatten = failwith "Not yet implemented"
```

Comment:
1. Include the type definition in this template. 
2. Include a placeholder for the function's argument.